src/contributors/ayushnanda21.md

```diff
@@ -0,0 +1,8 @@
+---
+name: "Ayush nanda"
+username: "ayushnanda21"
+image: "ayushnanda21"
```

Comment:
Hello @ayushnanda21 ,
Image field should be the complete name of the image including extension. In your case it should be
`image: ayushnanda21.jpg`